co-log-polysemy.cabal

```diff
@@ -31,14 +31,15 @@ tested-with:         GHC == 8.4.4
                      GHC == 8.6.5
                      GHC == 8.8.4
                      GHC == 8.10.7
-                     GHC == 9.0.1
+                     GHC == 9.0.2
+                     GHC == 9.2.3
 
 source-repository head
   type:                git
   location:            https://github.com/co-log/co-log-polysemy.git
 
 common common-options
-  build-depends:       base >= 4.11 && < 4.16
+  build-depends:       base >= 4.11 && < 4.17
```

Comment:
can't we just get rid of the upperbound ? updating haskell compiler is such a chore because of this.

Reply:
You are right - I did the same PR for ghc 9.4 now - but I wouldn't like to decide if we should remove upper bounds (there are pro's and con's). @vrom911 What do you think? Should I remove the upper bounds in https://github.com/co-log/co-log-polysemy/pull/16 to avoid the necessity to just adapt the meta-data of the package for every new GHC release in the future?